deploy/example_selflaunch.yaml

```diff
@@ -0,0 +1,11 @@
+# Refer to ../docs/webhost configuration sample.yaml
+
+# We'll be hosting VIA gunicorn
+SELFHOST: false
+SELFLAUNCH: true
+JOB_THRESHOLD: 0
+
+# Maximum concurrent world gens
+GENERATORS: 8
+
+HOSTERS: 5
```

Comment:
Default could be lower. I would default to a power of 2 just because it makes more sense on a multi-core system. So maybe 4?